client/src/pages/Apply/Apply.js

```diff
@@ -123,6 +102,7 @@ const Apply = ({ navigation }) => {
         description,
         iso: isoCode,
         glotto: glottoCode,
+        followUp,
```

Comment:
Nice, you added followUp to the `courseDetails`? Can you update our API so it saves this data in MongoDB. I believe you only need to update `api/src/models/language.js` by adding the followUp field to the `CourseDetails` Mongoose schema.